docs/notebooks/Comparing different embeddings with NucliaDB.ipynb

```diff
@@ -0,0 +1,587 @@
+    "\n",
+    "In this case we are using Instructor and MSMARCO\n",
+    "\n",
+    "Instructor is an LLM to which we can indicate with instructions the kind of embeddings we want to generate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 377,
+   "id": "fc7e55f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sentence_transformers import SentenceTransformer\n",
+    "model_marco = SentenceTransformer('sentence-transformers/msmarco-MiniLM-L6-cos-v5')\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 378,
+   "id": "2d93b876",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "load INSTRUCTOR_Transformer\n",
+      "max_seq_length  512\n"
+     ]
+    }
+   ],
+   "source": [
+    "from InstructorEmbedding import INSTRUCTOR\n",
+    "model_instructor = INSTRUCTOR('hkunlp/instructor-base')\n",
+    "instruction_query = \"Represent the question for retrieving relevant outdoors related posts:\"\n",
+    "instruction_posts= \"Represent the outdoors related post for retrieval:\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18fee1cb",
+   "metadata": {},
+   "source": [
+    "## Upload our data to NucliaDB\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 379,
+   "id": "e87813e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nucliadb_sdk.knowledgebox import KnowledgeBox\n",
+    "from nucliadb_sdk.utils import get_or_create"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 380,
+   "id": "26cf0d39",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_kb=get_or_create(\"my_outdoorsy_kb\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 381,
+   "id": "8495faec",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Vectorset is not created, we will create it for you\n",
+      "Vectorset is not created, we will create it for you\n"
+     ]
+    }
+   ],
+   "source": [
+    "for row in dataset[\"train\"]:\n",
+    "    my_kb.upload(\n",
+    "        text=row[\"upvoted_answer\"],\n",
+    "        vectors={\"ms-marco-vectors\": model_marco.encode([row[\"upvoted_answer\"]])[0],\n",
+    "                \"instructor-vectors\": model_instructor.encode([[instruction_posts,row[\"upvoted_answer\"]]])[0],\n",
+    "                }\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f99da942",
+   "metadata": {},
+   "source": [
+    "## Let's compare our results\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 382,
+   "id": "5efd9d92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def print_results(model_name, results):\n",
+    "    print(f\"---{model_name.upper()} RESULTS---\")\n",
+    "    count=0\n",
+    "    for result in results:\n",
+    "        if count>=3:\n",
+    "            break\n",
```

Comment:
NIT: might be more idiomatic like this:
```
for count, result in enumerate(results[:3])
```

Reply:
Updated, anyway I'll modify this and remove the helper function as soon as we can pass the max number of results we want as a parameter  in the sdk search